File: BA.Grisecorp.App.AspNetCore3/3 - Domain/BA.Grisecorp.App.AspNetCore3.API.Domain/Aggregates/GrupoEtapa/Grupo/Services/IGrupoEtapaService.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace BA.Grisecorp.App.AspNetCore3.API.Domain.Aggregates.GrupoEtapa.Grupo.Services
{
    public interface IGrupoEtapaService : IDisposable
    {
        int InserirGrupoEtapa(GrupoEtapa grupo);

        void AlterarGrupoEtapa(GrupoEtapa grupo);

        void DeletarGrupoEtapa(int id);

        GrupoEtapa ConsultarPorId(int id);

        List<GrupoEtapa> ListarGrupoEtapa();
    }
}

File: webapi/src/webapi/Domain/Aggregates/GrupoEtapa/GrupoEtapa/Services/IGrupoEtapaService.php
<?php
namespace webapi\Domain\Aggregates\GrupoEtapa\GrupoEtapa\Services;

interface IGrupoEtapaService
{
	public function InserirGrupoEtapa($grupoEtapa);
	public function AtualizarGrupoEtapa($grupoEtapa);
	public function DeletarGrupoEtapa($id_grupoetapa);
	public function ObterPorId($id_grupoetapa);
	public function ObterGrupoPorNome($nome);
	public function ListarGrupoEtapa();
}
?>